.github/actions/smoke-tests/action.yaml

```diff
@@ -43,10 +59,13 @@ runs:
         cache-from: type=gha,scope=${{ inputs.image }}
         cache-to: type=gha,scope=${{ inputs.image }},mode=max
         target: goreleaser
-        tags: 'docker.io/nginx/nginx-ingress:${{ inputs.image }}-${{ github.sha }}'
+        tags: 'docker.io/nginx/${{ steps.ingress-type.outputs.name }}:${{ inputs.image }}-${{ github.sha }}'
         load: true
         pull: true
         build-args: BUILD_OS=${{ inputs.image }}
+        secrets: |
+          "nginx-repo.crt=${{ inputs.nginx-crt }}"
```

Comment:
where do these certs come from ?

Reply:
`inputs.nginx-crt` and key are taking the input value from https://github.com/nginxinc/kubernetes-ingress/pull/2189/files#diff-b803fcb7f17ed9235f1e5cb1fcd2f5d3b2838429d4368ae4c57ce4436577f03fR178 and there are stored as secrets in GitHub like all the other secrets. Does that answer your question? I wasn't really sure which part you were asking about